src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin on-two($var) {
+  width: $var * 2.5;
```

Comment:
You can be more clear in variable naming, istead of just $var use something that describes the meening of this variable.